packages/pangea-sdk/src/main/java/cloud/pangeacyber/pangea/Client.java

```diff
@@ -26,33 +32,41 @@ protected Client(Config config, String serviceName) {
 		this.setCustomUserAgent(config.getCustomUserAgent());
 	}
 
-	protected HttpClient buildClient() {
-		java.net.http.HttpClient.Builder builder = HttpClient.newBuilder();
+	protected CloseableHttpClient buildClient() {
+		int timeout = 5000;
 		if (config.connectionTimeout != null) {
-			builder.connectTimeout(config.connectionTimeout);
+			timeout = (int) config.connectionTimeout.toMillis();
 		}
-		return builder.build();
+
+		RequestConfig config = RequestConfig
+			.custom()
+			.setConnectTimeout(timeout * 1000)
```

Comment:
is it ok to have the same timeout for these three?

Reply:
@dbenders not sure, but what relation you think we should we use?